.github/workflows/test.yml

```diff
@@ -12,8 +13,8 @@ jobs:
       fail-fast: false
       matrix:
         bazel_otp_name:
-        - "25"
-        - "26"
+          - "25"
+          - "26"
```

Comment:
I think this needs to be `25_3` and `26_2`

Reply:
Thanks!